src/index.ts

```diff
@@ -173,5 +214,8 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
  * @public
  */
 export function getInstance(): IEppoClient {
+  if (!EppoJSClient.instance) {
+    throw Error('init() must first be called to initialize a client instance');
+  }
```

Comment:
We do this in our Node SDK, seems like a more useful error than `Uncaught TypeError: Cannot read properties of undefined (reading 'instance')`.